--- a/debian/control
+++ b/debian/control
@@ -5,12 +5,11 @@ Maintainer: Bleemeo Packaging Team <packaging-team@bleemeo.com>
 Build-Depends: debhelper (>=9),
                dh-exec,
                dh-python,
-               dh-systemd,
-               python3-setuptools (>=30.3.0),
+               python3-setuptools,
                python3-all
 Standards-Version: 3.9.5
 Homepage: https://bleemeo.com
-X-Python3-Version: >= 3.4
+X-Python3-Version: >= 3.2
 
 Package: bleemeo-agent
 Architecture: all
--- a/debian/rules
+++ b/debian/rules
@@ -11,7 +11,7 @@ export PYBUILD_DESTDIR=debian/bleemeo-agent/
 
 # main packaging script based on dh7 syntax
 %:
-	dh $@ --with python3,systemd --buildsystem=pybuild
+	dh $@ --with python3 --buildsystem=pybuild
 
 # debmake generated override targets
 # Too cmplicated to provide examples here.
--- a/setup.cfg
+++ /dev/null
@@ -1,44 +0,0 @@
-[metadata]
-name = bleemeo-agent
-description = "Agent for Bleemeo"
-long_description = file: README.rst
-version = attr: bleemeo_agent.__version__
-author = Bleemeo
-author_email = hello@bleemeo.com
-url = https://bleemeo.com
-license = Apache 2.0
-keywords = monitoring, cloud, docker, aws
-classifiers =
-    Programming Language :: Python
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.4
-    Programming Language :: Python :: 3.5
-    Programming Language :: Python :: 3.6
-    License :: OSI Approved :: Apache Software License
-    Development Status :: 5 - Production/Stable
-    Intended Audience :: System Administrators
-
-[options]
-install_requires = 
-        apscheduler
-        jinja2
-        psutil >= 2.0.0
-        requests
-        six
-        pyyaml
-packages = find:
-include_package_data = True
-scripts =
-    bin/bleemeo-netstat
-
-[options.extras_require]
-docker = docker
-influxdb = influxdb > 2.6.0
-bleemeo = paho-mqtt
-sentry = raven
-web = flask
-
-[options.entry_points]
-console_scripts = 
-    bleemeo-agent = bleemeo_agent.core:main
-    bleemeo-agent-gather-facts = bleemeo_agent.facts:get_facts_root
--- a/setup.py
+++ b/setup.py
@@ -1,6 +1,42 @@
 #!/usr/bin/python
 
-from setuptools import setup
+from setuptools import find_packages, setup
+
+import bleemeo_agent
 
 
-setup()
+setup(
+    name="bleemeo-agent",
+    version=bleemeo_agent.__version__,
+    url='https://bleemeo.com',
+    license='Apache 2.0',
+    description="Agent for Bleemeo",
+    author='Bleemeo',
+    author_email='hello@bleemeo.com',
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[
+        'apscheduler',
+        'jinja2',
+        'psutil >= 2.0.0',
+        'requests',
+        'six',
+        'pyyaml',
+    ],
+    extras_require={
+        'docker': ['docker'],
+        'influxdb': ['influxdb > 2.6.0'],
+        'bleemeo': ['paho-mqtt'],
+        'sentry': ['raven'],
+        'web': ['flask']
+    },
+    scripts=(
+        'bin/bleemeo-netstat',
+    ),
+    entry_points = {
+        'console_scripts': [
+            'bleemeo-agent = bleemeo_agent.core:main',
+            'bleemeo-agent-gather-facts = bleemeo_agent.facts:get_facts_root',
+        ],
+    }
+)
